### PR TITLE
Xiaomi Mi Temperature & Humidity Monitor 2

### PR DESCRIPTION
**Pull request by smarts4home**
New model name in custom zigbee firmware - LYWSD03MMC-z for Xiaomi Mi Temperature & Humidity Monitor 2

### CODE REVIEW

File: src/devices/custom_devices_diy.ts [hunk 1]
```diff
@@ -1405,6 +1405,129 @@ const definitions: Definition[] = [
             }
         },
     },
+    {
+        // https://github.com/devbis/z03mmc/
```
Comment (devbis) [inline src/devices/custom_devices_diy.ts:1409]: It's not from this repo

File: src/devices/custom_devices_diy.ts [hunk 2]
```diff
@@ -1405,6 +1405,129 @@ const definitions: Definition[] = [
...
+            }),
+            binary({
+                name: 'enable_display',
+                valueOn: ['ON', 1],
+                valueOff: ['OFF', 0],
+                cluster: 'hvacUserInterfaceCfg',
+                attribute: {ID: 0x0011, type: dataType.boolean},
+                description: 'Whether to turn display on/off.',
+            }),
+            numeric({
```
Comment (devbis) [inline src/devices/custom_devices_diy.ts:1451]: Calibration settings are wrong for this fw

Comment (BenJamesAndo) [reply src/devices/custom_devices_diy.ts]: It seems this custom quirk is the current 'official' quirk: https://github.com/pvvx/ZigbeeTLc/blob/master/z2m/lywsd03mmc-z.js

**Comment by kafisc1** [comment]
This external converter (and also without any external convert) stopped working for my LYWSD03MMC-z since upgrading z2m with the following error:
Exception while calling fromZigbee converter: Definition 'LYWSD03MMC-z' has not endpoint defined}

Is anyone facing the same issue?

Firmware-Datum
20240211
Firmware-Version
3001-0120

External definiton:

const {battery, temperature, humidity} = require('zigbee-herdsman-converters/lib/modernExtend');

const definition = {
    zigbeeModel: ['LYWSD03MMC-z'],
    model: 'LYWSD03MMC-z',
    vendor: 'Xiaomi',
    description: 'Automatically generated definition',
    extend: [battery(), temperature({"endpointNames":["1"]}), humidity({"endpointNames":["1"]})],
    meta: {},
};

module.exports = definition;

**Pull request closed without merging**